app/components/search_result/no_result/filters_component.rb

```diff
@@ -3,7 +3,9 @@
 module SearchResult
   module NoResult
     class FiltersComponent < Blacklight::ConstraintsComponent
-      def query_constraints; end
```

Comment:
Do we want to display the original query now? I don't see that in the ticket, but maybe there was a conversation somewhere about it.

Reply:
... or was this to include adv search parameters, maybe? I don't know if that needs to be something upstream or if we should just own more of the logic here..

Reply:
my understanding is this was on purpose https://github.com/sul-dlss/SearchWorks/pull/4696#issuecomment-2808549092

Reply:
Okay I misunderstood what this comment was saying. This is fixed.